src/bioregistry/data/bioregistry.json

```diff
@@ -60327,7 +60331,8 @@
     "synonyms": [
       "ISBN-10",
       "ISBN-13"
-    ]
+    ],
+    "uri_format": "https://isbnsearch.org/isbn/$1"
```

Comment:
is this the PURL?

Are we not conflating URLs for humans and identifiers again?

Reply:
I don't think there is an issue here: this is simply a preferred override over the implicit imported resolver from identifiers.org (which is isbndb.com and is still there as one of the resolvers, just not the preferred one) which works better. Both of these are third-party providers of ISBN information, and I don't think either would be considered a "PURL". Is there a PURL for ISBN that should be added?